src/shared/components/results/__tests__/__snapshots__/SearchSuggestions.spec.tsx.snap

```diff
@@ -15,7 +15,7 @@ exports[`SearchSuggestions should render proteome suggestions given an organism
     </a>
      or restrict to reference proteome 
     <a
-      href="/uniprotkb?query=organism_id%3A9606+AND+%28proteome%3AUP000005640%29"
+      href="/uniprotkb?query=proteome%3AUP000005640"
```

Comment:
It seems the original query isn't going into the suggestions. For example, searching for `foo AND (organism_id:9606)`:

http://localhost:8080/uniprotkb?query=foo+AND+%28organism_id%3A9606%29

Then clicking the `restrict to reference proteome [UP000005640]` link goes to `proteome:UP000005640` while I would have expected `foo AND (organism_id:9606)` to be there.